### ISSUE TITLE
Error Upon Completion of Research

### ISSUE THREAD

**Issue by Xanarios** [Helfima/helmod#208]
Latest version available through the mod browser, 0.9.3, throws an error upon completion of research:

```
617.397 Error MainLoop.cpp:1199: Exception at tick 10222228: The mod Helmod: assistant for planning your base. caused a non-recoverable error.
Please report this error to the mod author.

Error while running event helmod::on_research_finished (ID 18)
__helmod__/controller/EventController.lua:131: attempt to call method 'resetCaches' (a nil value)
stack traceback:
	__helmod__/controller/EventController.lua:131: in function <__helmod__/controller/EventController.lua:129>
```

**Comment by Helfima** [comment]
oups i fix it tonight, I work
I saw a another bug in the recipe selector before go bed :)
use the previous version

**Comment by Helfima** [comment]
fixed in v0.9.4
